site/docs/flink.md

```diff
@@ -97,7 +97,21 @@ CREATE CATALOG hive_catalog WITH (
   'catalog-type'='hive',
   'uri'='thrift://localhost:9083',
   'clients'='5',
-  'property-version'='1'
+  'property-version'='1',
+  'hive-conf-dir'='/opt/hive/conf'
+);
+```
+
+We could also just specify the `warehouse` property (without specifying any hive configuration path) to initialize the hive catalog: 
```

Comment:
```suggestion
Alternatively one can instead set just the `warehouse` property (without specifying a Hive configuration directory) to initialize the Hive catalog: 
```

Reply:
Thanks for the change,  I will accept it. @massdosage  !